compiler/crates/relay-transforms/src/raw_text.rs

```diff
@@ -0,0 +1,93 @@
+use common::{ArgumentName, DiagnosticsResult, DirectiveName, Location, NamedItem, WithLocation};
+use graphql_ir::{
+    Argument, ConstantValue, Directive, OperationDefinition, Program, Transformed, Transformer,
+    Value,
+};
+use graphql_text_printer::{OperationPrinter, PrinterOptions};
+use intern::string_key::Intern;
+use lazy_static::lazy_static;
+
+use crate::test_operation_metadata::TEST_OPERATION_DIRECTIVE;
+
+lazy_static! {
+    pub static ref RAW_TEXT_DIRECTIVE_NAME: DirectiveName = DirectiveName("rawText".intern());
+    pub static ref RAW_TEXT_ARGUMENT_KEY: ArgumentName = ArgumentName("rawText".intern());
+}
```

Comment:
Doesn't need to define the value twice. This is just a perf optimization to initialize the string once.  The whole thing can be replaced with the  `intern!("rawText")` macro below.